pkg/ccpatch/ipnet/gen.go

```diff
@@ -0,0 +1,74 @@
+package ipnet
+
+// credits: https://github.com/korylprince/ipnetgen/tree/master
```

Comment:
I guess this will not be a problem of license because the crossplane code is open for everybody?
I this is fine due to licensing, wouldn't it make more sense to use directly the code from the repo as a dependency instead of copy pasting?

Reply:
I think it was done like this because he needed to add something to the implementation. The First() method.

Reply:
the original package is no longer maintained, as this logic is important for our generation business, i decided to copy it over and leave a note for the credits of the original maintainer.

As it's MIT Licensed feel free to remove this comment.

Reply:
We came to a conclusion that we can keep it since MIT is a very permissive license